WooCommerce/Classes/ViewRelated/Orders/Order Creation/StatusSection/OrderStatusSection.swift

```diff
@@ -54,6 +55,7 @@ private extension OrderStatusSection {
         enum StatusBadge {
             static let horizontalPadding: CGFloat = 12.0
             static let verticalPadding: CGFloat = 4.0
+            static let outsideVerticalPadding: CGFloat = 10.0
```

Comment:
nit: Should this be 16 instead of 10? The design handbook uses an 8pt grid and suggests 16 as the default for things like padding.